components/feature/qr/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+
+    <!-- Content description (not visible, for screen readers etc.): Description of an image view. -->
+    <string name="mozac_qr_scanner">QR scanner</string>
```

Comment:
nit: Following the style of the other strings that would be `mozac_feature_qr_scanner`.

Reply:
Thanks, updated!